final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +107,83 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+name: "Blarg",
+smell: "Sewage",
+weight: 7000,
+citiesDestroyed: ["Townsville", "Gotham", "Miami"],
+luckyNumbers: [4, 13, 6, 86],
+address: {"number" => 123, "street" => "sesame street", "State" => "New York",
+"Zip" => 12345}
+}
+
+puts scary_monster
+
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
 
+  class SuperHero
+    attr_accessor :name, :super_power, :age, :power_level
+
+    def initialize(name, super_power, age)
+      @name = name
+      @super_power = super_power
+      @age = age
+      @arch_nemesis = "The Syntax Error"
+      @power_level = 100
+      @energy_level = 50
+    end
+
+    def maximize_energy
+      @energy_level = 1000
+      puts "energy level replenished to #{@energy_level}"
+    end
+
+    def say_name
+        puts name
+    end
+
+    def gain_power(number)
```

Comment:
This method should increase the `power_level` by the `number` argument, ie. we would need to re-assign the `@power_level` variable to a value that is the sum of the current `@power_level` value and the `number` value.

Reply:
Tell me what you think of my amended method!